Replace generic man(1) instructions with Debian-specific information.
Indicate that the user needs to install the perl-doc package.

diff -Naur --exclude=debian perl-5.10.0.orig/pod/perl.pod perl-5.10.0/pod/perl.pod
--- perl-5.10.0.orig/pod/perl.pod	2007-11-18 02:26:25.000000000 +1100
+++ perl-5.10.0/pod/perl.pod	2007-11-25 23:41:20.000000000 +1100
@@ -220,32 +220,16 @@
     perlwin32		Perl notes for Windows
 
 
-By default, the manpages listed above are installed in the 
-F</usr/local/man/> directory.  
-
-Extensive additional documentation for Perl modules is available.  The
-default configuration for perl will place this additional documentation
-in the F</usr/local/lib/perl5/man> directory (or else in the F<man>
-subdirectory of the Perl library directory).  Some of this additional
-documentation is distributed standard with Perl, but you'll also find
-documentation for third-party modules there.
+On Debian systems, you need to install the B<perl-doc> package which
+contains the majority of the standard Perl documentation and the
+F<perldoc> program.
+
+Extensive additional documentation for Perl modules is available, both
+those distributed with Perl and third-party modules which are packaged
+or locally installed.
 
 You should be able to view Perl's documentation with your man(1)
-program by including the proper directories in the appropriate start-up
-files, or in the MANPATH environment variable.  To find out where the
-configuration has installed the manpages, type:
-
-    perl -V:man.dir
-
-If the directories have a common stem, such as F</usr/local/man/man1>
-and F</usr/local/man/man3>, you need only to add that stem
-(F</usr/local/man>) to your man(1) configuration files or your MANPATH
-environment variable.  If they do not share a stem, you'll have to add
-both stems.
-
-If that doesn't work for some reason, you can still use the
-supplied F<perldoc> script to view module information.  You might
-also look into getting a replacement man program.
+program or perldoc(1).
 
 If something strange has gone wrong with your program and you're not
 sure where you should look for help, try the B<-w> switch first.  It
